### PR TITLE
Feature/basic commands

### PR DESCRIPTION
**Pull request by StarBlasto**
gimme

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .env
+.venv
```
Comment (Cosmo517) [inline .gitignore]: You should not include __pycache__ within the git repository

File: Commands/Basic/help.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# /Commands/Basic/help.py
```
Comment (Cosmo517) [inline Commands/Basic/help.py]: You should use all lowercase folder names

File: Commands/Gambling/blackjack.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# /Commands/Gambling/blackjack.py
```
Comment (Cosmo517) [inline Commands/Gambling/blackjack.py]: +1 for folder names

**Comment by StarBlasto** [comment]
fixed issues

File: .gitignore (hunk 1 above)
Comment (StarBlasto) [reply .gitignore]: fixed

File: Commands/Basic/help.py (hunk 2 above)
Comment (StarBlasto) [reply Commands/Basic/help.py]: fixed

**Pull request merged**